app/controllers/judgements_controller.rb

```diff
@@ -14,6 +15,9 @@ def show
                        .order(sent_at: :desc).page(params[:page]).decorate
     @entry_requests = @evaluation.entry_requests.reject { |er| er.entry_type == EntryRequest::KDO }
     @entry_requests = EntryRequestDecorator.decorate_collection @entry_requests
+    @evaluated_memberships =
+      @evaluation.point_requests.map     { |pr| pr.user.membership_for(@evaluation.group) }
+                                .sort_by { |pr| pr.user.decorate.full_name }
```

Comment:
I think it's a bit weird to decorate here, and then in the view, but I could not find a better solution.

Reply:
Yes, this query is a bit odd. If you check [here in the view](https://github.com/kir-dev/pek-next/blob/af5c24c52c81c70efb11bdc549029b27b898d46e/app/views/judgements/_point_requests.html.erb#L34L54) for displaying the table only `@point_details` and a list of users are needed.

For the [next summary part](https://github.com/kir-dev/pek-next/blob/af5c24c52c81c70efb11bdc549029b27b898d46e/app/views/judgements/_point_requests.html.erb#L64L93) it seems like the memberships are required, but actually not. [Here](https://github.com/kir-dev/pek-next/blob/af5c24c52c81c70efb11bdc549029b27b898d46e/app/helpers/judgements_helper.rb#L31) in the `JudgementHelper` you can rewrite the `membership.user` to `user` and pass the `@user` array as an argument for each methods.

So actually you could decrease the complexity in the controller to this:
```ruby
@users = @evaluation.point_requests
                    .include(:user)
                    .map(&:user)
                    .decorate_collection
                    .sort_by(&:full_name)
```
or this
```ruby
user_ids = @evaluation.point_requests.pluck(:user_id)
@users   = UserDecorator.where(id: user_ids).sort_by(&:full_name)
```

Reply:
I rewrote the methods in `JudgementHelper`, it made the code more straightforward. I could not call `decorate_collection` on the User array, I had to decorate it in a more verbose way.

Reply:
Ah, you are right with the `decorate_collection`, probably the map returns an `Array`.

In the case of the second one you should include the `decorates_finders` in the `UserDecorator` to make it work. Btw I see that the `full_name` method is defined on the `User` not on the decorator, so it's not required to decorate the collection before sorting.

I have a new idea also, you can move the sorting to database level with something similar:
```ruby
# /app/models/user.rb
scope :order_by_name, -> { order(:lastname).order(:firstname) }

# /app/controllers/judgements_controller.rb
user_ids = @evaluation.point_requests.pluck(:user_id)
@users   = UserDecorator.where(id: user_ids).order_by_name
```
Though it's loosely connected to the essence of this PR, so if you prefer let's create an issue on that.

Reply:
I tried out the suggested modifications.
`UserDecorator.where ` does not return decorated Users even after including  decorates_finders in the User model. But `find` does work:
```ruby
@users   = UserDecorator.find(user_ids)
```
Another problem is If I define `order_by_name` in the model, the decorator can't find it and throws undefined exception. 
I have mixed feeling about making queries with decorators. The code is batter to look at, but it feels like its not the decorators responsibility. 
So I created issue #251 to solve these problems later.